tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Iast.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="ConfigurationKeys.Iast.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Configuration
+{
+    internal partial class ConfigurationKeys
+    {
+        internal class Iast
+        {
+            /// <summary>
+            /// Configuration key for enabling or disabling the IAST.
+            /// Default is value is false (disabled).
+            /// </summary>
+            public const string Enabled = "DD_IAST_ENABLED";
+
+            /// <summary>
+            /// Configuration key for enabling or disabling the Weak hash algorithms detection.
+            /// Default is value is true (enabled).
+            /// </summary>
+            public const string WeakHashAlgorithms = "DD_IAST_WEAK_HASH_ALGORITHMS";
```

Comment:
Should we add an `ENABLED` suffix: `DD_IAST_WEAK_HASH_ALGORITHMS_ENABLED`?

Not sure if this setting is shared across all libraries, but we have generally tried to use the `ENABLED` suffix for Booleans that turns features on and off.

Reply:
Ahh, thanks for the link! It looks like it's the comment above that is wrong:
```
Default is value is true (enabled).
```

This setting accepts a list of strings, default: `"MD2, MD4, MD5, RIPEMD128, SHA1"`.